calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/transformation_estimator.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MARKER_RADAR_LIDAR_CALIBRATOR__TRANSFORMATION_ESTIMATOR_HPP_
+#define MARKER_RADAR_LIDAR_CALIBRATOR__TRANSFORMATION_ESTIMATOR_HPP_
+
+#include <Eigen/Dense>
+#include <autoware/universe_utils/geometry/geometry.hpp>
+#include <marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <tf2_eigen/tf2_eigen.hpp>
+
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
+
+#include <ceres/ceres.h>
+#include <pcl/pcl_base.h>
+#include <pcl/point_types.h>
+#include <pcl/registration/transformation_estimation_svd.h>
+#include <tf2/convert.h>
+
+namespace marker_radar_lidar_calibrator
+{
+
+class TransformationEstimator
+{
+public:
+  using PointType = pcl::PointXYZ;
```

Comment:
This error was before but can you make all instances of PointType be declared only once ?
(currently it is defined three times, and there is the risk of changing one and forgetting the others - quite low though)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/002d87c59f4884162e45abcbd9668ea9258edb68